src/options.tsx

```diff
@@ -60,13 +70,23 @@ function BasicSettings() {
         </label>
 
         <input
-          className="bg-gray-50 border w-64 border-gray-300 text-gray-900 text-sm rounded-lg 
+          className="bg-gray-50 border w-64 border-gray-300 text-gray-900 text-sm rounded-lg
               focus:ring-blue-500 focus:border-blue-500 block"
           value={apiURL}
           onChange={updateApiURL}
           id="api_url"
         />
       </div>
+      <div className="flex flex-col gap-y-2">
+        <label htmlFor="api_url" className="text-xl font-medium">
+          Filter Rule
```

Comment:
I think it's better to have a short description to explain what's filter rule for

Reply:
![CleanShot 2023-12-13 at 00 14 12@2x](https://github.com/MichaelYuhe/ai-group-tabs/assets/44605072/d3c759b8-fbeb-440f-9ae0-c87b49dfde82)

Do you think my explanation will be clearer like this?

Reply:
yes! it's much better